scripts/library/htpasswd.sh

```diff
@@ -23,7 +18,16 @@ htpasswd_add_user(){
   touch "${HTPASSWD_FILE}" "${HTPASSWD_FILE}".txt
   sed -i '/# '"${USER}"'/d' "${HTPASSWD_FILE}".txt
   echo "# ${USER} - ${PASS}" >> "${HTPASSWD_FILE}.txt"
-  htpasswd -bB -C 10 "${HTPASSWD_FILE}" "${USER}" "${PASS}"
+
+  if which htpasswd >/dev/null 2>&1; then
+    echo "using local htpasswd..."
+    htpasswd -b -B -C10 "${HTPASSWD_FILE}" "${USER}" "${PASS}"
+  else
+    echo "using oc to run pod..."
+    oc run \
```

Comment:
it may make sense to just have this always be the behavior in the automation, to reduce complexity